com.vrcfury.vrcfury/Editor/VF/Feature/ArmatureLinkBuilder.cs

```diff
@@ -493,6 +494,10 @@ private class Links {
                                                     "to point to the corresponding bone on the base armature. This is extremely unusual. Don't use this " +
                                                     "unless you know what you are doing."));
             adv.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("physbonesOnAvatarBones")));
+            adv.Add(new VisualElement { style = { paddingTop = 10 } });
+            adv.Add(VRCFuryEditorUtils.WrappedLabel("Allow avatar physbones to affect new merged bones (Skin Rewrite only):"));
+            adv.Add(VRCFuryEditorUtils.WrappedLabel("If checked, physbones in the avatar will affect the new bones merged into the armature."));
+            adv.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("physbonesForNewBones")));

```

Comment:
In what case is this wanted? Typically doing this will switch the physbone into multi-child mode, which will do strange things to not only the added bone, but also the existing parent. Is there a case where this would be better than adding a new physbone for the added child?